salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -22,3 +27,31 @@ edx:
           - edx-{{ app_type }}
 {% endfor %}
 {% endfor %}
+
+edx-worker:
+  {% set num_subnets = subnetids|length %}
+  {% for app_type, num_instances in app_types.items() %}
+  {% for id_num in range(num_instances) %}
```

Comment:
This needs to reference the edx-worker. e.g. `for id_num in range(num_instances.edx-worker)`, same for edxapp above.